lib/serverless-tools/deployer/ecr_pusher.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+require "aws-sdk-ecr"
+
+module ServerlessTools
+  module Deployer
+    class EcrPusher
+      def initialize(client:, git:, config:)
+        @client = client
+        @git = git
+        @config = config
+      end
+
+      def push(local_image_name:)
+        if image_tags.include?(tag)
+          puts "Did not upload #{tagged_image_uri} as it already exists!"
+        else
+          system("docker tag #{local_image_name} #{tagged_image_uri}")
+          system("docker push #{tagged_image_uri}")
```

Comment:
Do we need to do anything for the github actions to have docker installed - or do they already have it available?

Reply:
It's already available 🪄  We've been successfully issuing Docker commands directly in the workflows here: https://github.com/fac/company-engagement-health-score/pull/10/files#diff-25048be876abab7f9c933cddacfac72d5692c5589e61f6b6a7302260a10a4d28R37

Reply:
Magic!